test/sequential/test-http-keepalive-lifo-sockets.js

```diff
@@ -0,0 +1,95 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const http = require('http');
+
+// To test the LIFO behavior of the agent:
+// Send a request and store the local TCP port
+// number used to send a request to the server.
+
+// Store a list of port numbers used to process requests...
+let port_list = [];
+
+const server = http.createServer(function(req, res) {
+  // Return the remote port number used for this connection.
+  res.end(req.socket.remotePort.toString(10));
+});
```

Comment:
Can you add a few common.mustCall() in a few critical placesv

Reply:
Sure